awx/main/management/commands/create_oauth2_token.py

```diff
@@ -0,0 +1,26 @@
+from awx.api.serializers import OAuth2TokenSerializer
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+
+class Command(BaseCommand):
+    """Command that creates an OAuth2 token for a certain user. Returns the value of created token."""
+    help='Usage: awx-manage create_oauth2_token --user=username. Will return the value of created token.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--user', dest='user', type=str)
+
+    def handle(self, *args, **options):
+        if not options['user']:
+            self.stdout.write("Username not supplied. Usage: awx-manage create_oauth2_token --user=username.")
+            return
+        try:
+            user = User.objects.get(username=options['user'])
+        except ObjectDoesNotExist:
+            self.stdout.write("The user does not exist.")
+            return
+        config = {'user': user}
+        serializer_obj = OAuth2TokenSerializer()
+        token_record = serializer_obj.create(config, True)
```

Comment:
Is there a default scope when you create a token this way (we probably want it to be `write` by default)?

Reply:
There is currently no default for scope.  Though that is not a bad idea.  In the case of this awx-manage command, we definitely want `write` as the default.  This should be added in the `create_oauth2_token` command.

Reply:
awx.api.serializers.OAuth2TokenSerializer.create() will make the token write by default.

Reply:
There is currently no default for scope on the model. Though that is not a bad idea.  In the case of this awx-manage command, we definitely want write as the default. This should be added in the create_oauth2_token command.